recirq/quantum_chess/circuit_transformer.py

```diff
@@ -22,19 +22,31 @@
 
 
 class CircuitTransformer:
+    """Abstract interface for circuit transformations.
+
+    For example: named -> grid qubit transformations.
+    """
+    def __init__(self):
```

Comment:
nit: should we add `device` as a parameter?

Reply:
IMO there's not much benefit to have the base class take the device and do nothing else with it besides storing a data member, even if all the implementations will end up doing this. If the implementations end up sharing some common functionality, there will probably be a cleaner way to share that.

Reply:
sgtm 👍